static/app/views/profiling/profileSummary/index.tsx

```diff
@@ -273,12 +280,54 @@ function ProfileSummaryPage(props: ProfileSummaryPageProps) {
             query={query}
             hideCount
           />
+          <ProfileVisualizationContainer>
+            <ProfileVisualization>
+              <ProfileGroupProvider
+                type="flamegraph"
+                input={data ?? null}
+                traceID=""
+                frameFilter={undefined}
+              >
+                <FlamegraphStateProvider
+                  initialState={{
+                    preferences: {
+                      sorting: 'alphabetical',
+                    },
+                  }}
+                >
+                  <FlamegraphThemeProvider>
+                    <AggregateFlamegraph
+                      hideSystemFrames={false}
+                      setHideSystemFrames={() => void 0}
```

Comment:
Not wiring anything up yet, just making sure we fetch data and render some basics first